src/patternfly/components/Page/page.scss

```diff
@@ -34,20 +34,27 @@
 
   // Header nav
   --pf-c-page__header-nav--PaddingLeft: var(--pf-global--spacer--md);
-  --pf-c-page__header-nav--md--PaddingLeft: var(--pf-global--spacer--xl);
+  --pf-c-page__header-nav--md--PaddingLeft: var(--pf-global--spacer--lg);
   --pf-c-page__header-nav--lg--PaddingLeft: 0;
   --pf-c-page__header-nav--lg--MarginLeft: var(--pf-global--spacer--xl);
   --pf-c-page__header-nav--lg--MarginRight: var(--pf-global--spacer--xl);
-  --pf-c-page__header-nav--BackgroundColor: var(--pf-global--BackgroundColor--dark-transparent-100);
+  --pf-c-page__header-nav--BackgroundColor: #242424; // Custom color for this element only
   --pf-c-page__header-nav--lg--BackgroundColor: transparent;
```

Comment:
Do we need to set this variable, could you change the breakpoint to max-width and just set the background color there. I know its a breaking change but maybe we could create an issue to remove it?

Reply:
Doesn't matter to me. It's probably the same amount of code either way - you're losing a variable but adding a media query. This way is technically more flexible, too, not that we need it to be. Feel free to create an issue if you think we should remove it.

Reply:
Thats ok then